rebuild/src/addon.cc

```diff
@@ -56,12 +56,28 @@ BlstTsAddon::BlstTsAddon(Napi::Env env, Napi::Object exports)
     SecretKey::Init(env, exports, this);
     PublicKey::Init(env, exports, this);
     Signature::Init(env, exports, this);
-    // Functions::Init(env, exports);
+    Functions::Init(env, exports);
     env.SetInstanceData(this);
 }
 
 std::string BlstTsAddon::GetBlstErrorString(const blst::BLST_ERROR &err) {
     return _blst_error_strings[err];
 }
 
+bool BlstTsAddon::GetRandomBytes(blst::byte *bytes, size_t length) {
```

Comment:
why can't we use node crypto's lib use `CSPRNG`, also what about seeding of the random generator as has been noted in https://github.com/nodejs/node/blob/4166d40d0873b6d8a0c7291872c8d20dc680b1d7/src/crypto/crypto_random.cc#L162

Reply:
I tried again but after a very deep dig these are the headers that are exported by node.

![Screenshot 2023-07-10 at 8 56 03 PM](https://github.com/ChainSafe/blst-ts/assets/18608739/6291172f-c9c9-483e-93a7-f12a283879bf)

This is the line that makes `node_crypto.h` internal:
https://github.com/nodejs/node/blob/4166d40d0873b6d8a0c7291872c8d20dc680b1d7/src/node_crypto.h#L25

Even with setting the flag `NODE_WANT_INTERNALS` its still unavailable.  The symbols are not exported so it doesn't build.

I tried bringing in just the headers manually to statically link `crypto_util.h` but there are a bunch of other internal headers that it calls and it started to grow unwieldy.

I added a check at module startup that makes sure the PRNG is seeded and working correctly and added the check after each call to make sure they were successful.

I am open to suggestions though if there is another direction you think we should go.

Reply:
alright, lets go with current impl